### DISCUSSION TITLE
How to present all Cell Voltages in VRM?

### DISCUSSION THREAD

**Discussion by hansgeorgk** [Louisvdw/dbus-serialbattery#491] [Q&A] [answered]
Hi,

I am seeing large differences in min/max cell values while charging and discharging. Even causing the system to shut down :-( 
I want to verify it is a single cell (cell 4) going crazy. 
![Bildschirm­foto 2023-03-10 um 08 58 13](https://user-images.githubusercontent.com/100286224/224257407-bff2ac36-ce64-41e4-827e-61e43f186361.png)
![Bildschirm­foto 2023-01-17 um 06 54 16](https://user-images.githubusercontent.com/100286224/224254667-9be6d4f3-0c9f-4458-a463-51071b8c3d1a.png)
since the individual cell voltages are available in in MQTT / Remote console, I would like to get them into VRM to get a long time view on the voltages.

Can you give me some hints how to achieve this?
thanks
cheers
HG

**Comment by mr-manuel** [comment, marked as answer]
Hello,

since displaying the cell voltages is a feature of this driver, it's currently not possible to log them in VRM. However there are other possibilities. Probably this can help you https://github.com/Louisvdw/dbus-serialbattery/issues/473#issuecomment-1440253828
